core/src/com/mygdx/crazybomber/MainMenu.java

```diff
@@ -0,0 +1,121 @@
+//TODO: player set name in order to progress to picking a room
+package com.mygdx.crazybomber;
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+public class MainMenu implements Screen {
+    private Stage stage;
+    private Table table;
+    private TextButton buttonPlay, buttonExit;
+    private Label heading;
+    private Skin skin;
+    private TextureAtlas atlas;
+    private Sprite background;
+    private SpriteBatch batch;
+    private MapRepository mapRepository;
+    private RoomRepository roomRepository;
+    private StateRepository stateRepository;
+    private TextField usernameTextField;
+
+    public MainMenu(MapRepository mapRepository, RoomRepository roomRepository, StateRepository stateRepository) {
+        this.mapRepository = mapRepository;
+        this.roomRepository = roomRepository;
+        this.stateRepository = stateRepository;
+    }
+    @Override
+    public void show() {
+        final MainRooms mainRooms = new MainRooms(mapRepository, roomRepository, stateRepository);
+
+        batch = new SpriteBatch();
+        background = new Sprite(new Texture(("background.jpg")));
+        background.setSize(Gdx.graphics.getWidth(),Gdx.graphics.getHeight());
+
+        stage = new Stage();
+        Gdx.input.setInputProcessor(stage);
+        atlas = new TextureAtlas("ui/button.pack");
+        skin = new Skin(Gdx.files.internal("menuSkin.json"),atlas);
+        table = new Table(skin);
+        table.setBounds(0,0,Gdx.graphics.getWidth(),Gdx.graphics.getHeight());
+
+        usernameTextField = new TextField("YEEAA BOIIIIIIIIIIIII", skin);
```

Comment:
does this fextfield work? if not take it out. Again, not sure if this is supposed to be textField, thought it was supposed to be a textArea.